src/Support/helpers.php

```diff
@@ -7,6 +7,7 @@
      * Create a SemVer version object.
      *
      * @return \PHLAK\SemVer\Version
+     * @throws \PHLAK\SemVer\Exceptions\InvalidVersionException
      */
```

Comment:
`@throws` before `@return` to pass CI.

```suggestion
     * Create a SemVer version object.
     *
     * @throws \PHLAK\SemVer\Exceptions\InvalidVersionException
     *
     * @return \PHLAK\SemVer\Version
     */
```